### PR TITLE
Revert option implemented

### PR DESCRIPTION
**Pull request by andreasonny83**
Implemented revert option to swap the filename with the hash in the filename generated

### CODE REVIEW

File: tasks/wp-rev.js [hunk 1]
```diff
@@ -35,12 +36,19 @@ module.exports = function (grunt) {
                 var hash = md5(f, options.algorithm, 'hex'),
                 prefix = hash.slice(0, options.length),
                 filename = path.basename(f),
-                renamed = [prefix, filename].join('.'),
-                outPath = path.resolve(path.dirname(f), renamed),
-                regex = new RegExp('([0-9a-f]*\\.)*?' + filename),
+                ext = path.extname(filename),
+                baseName = path.basename(filename, ext),
+                renamed;
+                if ( options.revert ) {
```
Comment (Melindrea) [inline tasks/wp-rev.js]: Please remove the spaces before and after options.revert. After that I'll merge it in =)

**Comment by andreasonny83** [comment]
Done,
Thank you

File: tasks/wp-rev.js (hunk 1 above)
Comment (andreasonny83) [inline tasks/wp-rev.js]: I made the change, thank you

**Comment by Melindrea** [comment]
And now it's on NPM

**Comment by andreasonny83** [comment]
Great!

**Pull request merged**